packages/auth/package.json

```diff
@@ -137,8 +137,8 @@
     "@firebase/app": "0.10.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-strip": "2.1.0",
-    "@types/express": "4.17.17",
-    "chromedriver": "116.0.0",
+    "@types/express": "4.17.21",
+    "chromedriver": "119.0.1",
```

Comment:
Woot, I guess we can go ahead and bump this to the latest now that we're on Node 20